src/EditorFeatures/Core.Wpf/QuickInfo/Presentation/QuickInfoPresenter.cs

```diff
@@ -24,7 +24,8 @@ internal partial class QuickInfoPresenter : ForegroundThreadAffinitizedObject, I
         private readonly DeferredContentFrameworkElementFactory _elementFactory;
 
         [ImportingConstructor]
-        public QuickInfoPresenter(IQuickInfoBroker quickInfoBroker, DeferredContentFrameworkElementFactory elementFactory)
+        public QuickInfoPresenter(IThreadingContext threadingContext, IQuickInfoBroker quickInfoBroker, DeferredContentFrameworkElementFactory elementFactory)
```

Comment:
No `[Obsolete]` attribute on this importing constructor?

Reply:
➡️ I haven't added the attributes to all the old MEF types yet. I went ahead and added it to ones I saw in this pull request that weren't referenced by test code, but left the ones that are still called by test code that hasn't been updated. It's a situation where we want all new code to follow the new pattern, but old code is getting there over time.

Reply:
➡️ Filed https://github.com/dotnet/roslyn/issues/29182